src/teleop/gui/src/components/AutonTask.vue

```diff
@@ -153,27 +169,31 @@ export default {
     watch: {
         // Publish auton LED color to ESW
         nav_state_color: function (color) {
-            let ledMsg = {
-                type: 'AutonLed',
-                color: 'Null'
-            }
             if (color == navBlue) {
-                ledMsg.color = 'Blue'
-                this.greenHook = false
+                this.ledColor = 'blue'
             } else if (color == navRed) {
-                ledMsg.color = 'Red'
-                this.greenHook = false
-            } else if (color == navGreen && !this.greenHook) {
-                ledMsg.color = 'Green'
-                this.greenHook = true //Accounting for the blinking between navGrey and navGreen
-            }
-            if (!this.greenHook || ledMsg.color == 'Green') {
-                // TODO: Implement this once ESW has this interface back up
-                // this.publish('/auton_led',ledMsg)
+                this.ledColor = 'red'
+            } else if (color == navGreen) {
+                this.ledColor = 'green'
+            } else if (color == navGrey) {
+                this.ledColor = 'off'
             }
```

Comment:
Could potentially call sendColor here()

Reply:
On second thought it may also be nice to not send the service when blinking between green and off. Instead ESW could just know that when it's green it's supposed to blink. I'm not super concerned about this though

Reply:
I asked Sashreek and he said to do it this way and it would just make his job easier. @tabiosg What are your thoughts on this?

Reply:
@sashreek1 i already code for the science nucleo to deal with blinking. it should NOT be this way unless this node gets transferred into the jetson end

Reply:
just send green. it should be fine.

Reply:
Also, when is the case where we would the color would ever be green or Off? I feel like it should always be in teleop, auton, or done state.

Reply:
I had thought that done state would flash green on and off.

Reply:
Sorry, i made a typo. I meant gray or Off. I noticed that ur gui code had the following:

else if (color == navGrey) {
                this.ledColor = 'off'
            }